docs/News.md

```diff
@@ -1,3 +1,49 @@
+## July 29, 2024
+*   **Added data** consisting of 12,863 samples from 33 TCGA and 10 CPTAC studies from the GDC as part of the NCI-CRDC initiative. The studies are generated though the [ISB-CGC](https://portal.isb-cgc.org/) BigQuery tables.
```

Comment:
Should we expand the acronyms and/or link out, in case users aren't familiar with them?

Reply:
Also, Ben wants to link out directly to the subpage with BigQuery tables rather than the landing page of the ISB-CGC site.

```suggestion
*   **Added data** consisting of 12,863 samples from 33 TCGA and 10 CPTAC studies from the [Genomic Data Commons (GDC)](https://gdc.cancer.gov) as part of the [Cancer Research Data Commons (NCI-CRDC)](https://datacommons.cancer.gov) initiative. The studies are generated through the [ISB-CGC BigQuery tables](https://bq-search.isb-cgc.org/search?status=current).
```

Reply:
Thanks for the review. Updated.